packages/fiori/src/Wizard.ts

```diff
@@ -251,7 +176,100 @@ const metadata = {
  * @appenddocs sap.ui.webc.fiori.WizardStep
  * @public
  */
+@customElement("ui5-wizard")
+@languageAware
+@fastNavigation
+
+/**
+ * Fired when the step is changed by user interaction - either with scrolling,
+ * or by clicking on the steps within the component header.
+ *
+ * @event sap.ui.webc.fiori.Wizard#step-change
+ * @param {WizardStep} step The new step.
+ * @param {WizardStep} previousStep The previous step.
+ * @param {boolean} changeWithClick The step change occurs due to user's click or 'Enter'/'Space' key press on step within the navigation.
+ * @public
+ */
+@event("step-change", {
+	detail: {
+		step: { type: WizardStep },
+		previousStep: { type: WizardStep },
+		changeWithClick: { Boolean },
+	},
+})
+
 class Wizard extends UI5Element {
+	/**
+	 * Defines the width of the <code>ui5-wizard</code>.
+	 * @private
+	 */
+	@property({ validator: Float })
+	width?: number
+
+	/**
+	 * Defines the threshold to switch between steps upon user scrolling.
+	 * <br><br>
+	 *
+	 * <b>For Example:</b>
+	 * <br>
+	 * (1) To switch to the next step, when half of the step is scrolled out - set <code>step-switch-threshold="0.5"</code>.
+	 * (2) To switch to the next step, when the entire current step is scrolled out - set <code>step-switch-threshold="1"</code>.
+	 *
+	 * <br><br>
+	 * <b>Note:</b> Supported values are between 0.5 and 1
+	 * and values out of the range will be normalized to 0.5 and 1 respectively.
+	 * @private
+	 * @type {sap.ui.webc.base.types.Float}
+	 * @defaultvalue 0.7
+	 * @since 1.0.0-rc.13
+	 */
+	@property({ validator: Float, defaultValue: STEP_SWITCH_THRESHOLDS.DEFAULT })
+	stepSwitchThreshold!: number;
+
+	/**
+	 * Defines the height of the <code>ui5-wizard</code> content.
+	 * @private
+	 */
+	@property({ validator: Float })
+	contentHeight?: number;
+
+	@property({ multiple: true })
```

Comment:
We have to specify the type of the property otherwise it will be treated as string. In that case property will be treated as Array<string>

`{type: Object, multiple: true }`